apps/explorer/src/hooks/useImageMod.ts

```diff
@@ -19,23 +34,18 @@ export function useImageMod({
     url?: string;
     enabled?: boolean;
 }) {
+    const { request } = useAppsBackend();
+
     return useQuery({
-        queryKey: ['image-mod', url, enabled],
+        queryKey: ['image', url, enabled],
         queryFn: async () => {
-            if (!isURL || !enabled) return true;
-            try {
-                const resp = await fetch(`https://imgmod.sui.io/img`, {
-                    method: 'POST',
-                    body: JSON.stringify({ url }),
-                    headers: { 'content-type': 'application/json' },
-                });
-                const allowed = await resp.json();
-                return allowed;
-            } catch (e) {
-                return false;
-            }
+            if (!isURL || !enabled) return placeholderData;
```

Comment:
Can you also potentially filter this by supported content type? I think we do a content type lookup _somewhere_, but don't want to hit the API for content types that aren't supported.